README.md

```diff
@@ -78,7 +78,7 @@ The `build` target is split into two sub-targets:
 
 When building from the official source tarballs which include pre-built frontend files, the `frontend` target will not be triggered, making it possible to build without Node.js and Internet connectivity.
 
-Parallelism (`make -j <num>`) is not supported.
+Parallelism (`make -j<num>`) is not supported.
```

Comment:
```suggestion
Parallelism (`make -j <num>`) is not supported.
```

According to `make --help`: `  -j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.`

The space is correct and documented by the manual (while no space is also fine)